### DISCUSSION TITLE
https://plugin-search-pro.vuejs.press/zh/guide.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#691] [Announcements]
# https://plugin-search-pro.vuejs.press/zh/guide.html

vuepress-plugin-search-pro 是一个强大的客户端搜索插件，可以为你的站点添加自定义索引与全文搜索功能。 快速上手 索引范围 默认情况下，插件仅索引标题，文章摘要和你添加的自定义字段。如果你想要索引文章的全部内容，你可以通过设置 indexContent: true 来开启。 当索引不基于单词的语言时，例如中文、日语或韩语，你需要...

https://plugin-search-pro.vuejs.press/zh/guide.html

**Comment by 006hjy** [comment]
如何让搜索区分大小写字母？现在默认是不区分大小写的。

**Reply by Mister-Hope** [reply]
搜素正常就应该不区分大小写，不会考虑这个选项。

**Comment by oneao** [comment]
博主，我用的vuepress-theme-reco2，然后配置如下
```ts
plugins: [
        searchProPlugin({
            // 配置选项
            indexContent: true,
            //autoSuggestions: false, //为什么我加了这个配置项，键盘的上下移动键选中就无法生效。
            queryHistoryCount: 0,
            resultHistoryCount: -1
        }),
    ],
```

**Reply by Mister-Hope** [reply]
回头看看，你可以提一个issue。

可能是键盘监听器那里有点问题

**Comment by rosa5500** [comment]
如何通过组件的方式更灵活的引用？

**Comment by Web-NiuBi** [comment]
试着问下 customFields可以对不同项进行图标定制吗 不想统一都是❤
